js/testing.js

```diff
@@ -0,0 +1,61 @@
+// Testing.js
+
+(function(){
+
+	var newTestingDiv;
+
+function test_group(name, _callback){
+
+	var testingDiv = document.getElementById("testingDiv");
+
+	newTestingDiv = document.createElement("div");
+	newTestingDiv.className = "row col-md-12";
+	newTestingDiv.style.marginTop = "5%";
+	testingDiv.appendChild(newTestingDiv);
+
+	_callback();
+
+	newTestingDiv.childNodes.forEach(function(currAssert){
```

Comment:
forEach works, but you should use for-of if supported (ES6), forEach is bad for memory.